config/src/main/java/com/alibaba/nacos/config/server/aspect/RequestLogAspect.java

```diff
@@ -67,7 +66,7 @@
     public Object interfacePublishSingle(ProceedingJoinPoint pjp, HttpServletRequest request,
                                          HttpServletResponse response, String dataId, String group, String tenant,
                                          String content) throws Throwable {
-        final String md5 = content == null ? null : Md5Utils.getMD5(content, Constants.ENCODE);
+        final String md5 = content == null ? null : MD5.getInstance().getMD5String(content);
```

Comment:
I think the MD5 implementation will not be as good as the MD5Utils you delete because of the locking mechanism, you can only modify the string method of MD5Utils

old

```
return new BigInteger(1, messageDigest.digest(bytes)).toString(HEX_VALUE_COUNT);
```

new

```java
public String bytes2string(byte[] bt) {
        int l = bt.length;

        char[] out = new char[l << 1];

        for (int i = 0, j = 0; i < l; i++) {
            out[j++] = digits[(0xF0 & bt[i]) >>> 4];
            out[j++] = digits[0x0F & bt[i]];
        }

        return new String(out);
    }
```

Reply:
Thanks for your patient.I choose this one ,consider MessageDigest classes  is instantiated only once.